app/components/join-steps/status-card.js

```diff
@@ -1,13 +1,24 @@
-import { action } from '@ember/object';
 import Component from '@glimmer/component';
 import { inject as service } from '@ember/service';
+import { action } from '@ember/object';
+import { tracked } from '@glimmer/tracking';
 import { APPLICATION_STATUS_TYPES } from '../../constants/join';
+import { USER_JOINED_LINK } from '../../constants/apis';
+import { TOAST_OPTIONS } from '../../constants/toast-options';
+import { ANKUSH_TWITTER } from '../../constants/urls';
 
 export default class StatusCardComponent extends Component {
   @service login;
   @service router;
+  @service onboarding;
+  @service toast;
+
+  @tracked joinLink = USER_JOINED_LINK(this.login.userData?.id);
```

Comment:
If undefined is passed here?
won't joinLink be invalid?

Reply:
Yes as this is join link
  return `${APPS.HOME}/intro?id=${userId}&status=submitted`;


so either we can do something like this what your suggestion
  @action
  initializeJoinLink() {
    const userId = this.login.userData?.id;
    this.joinLink = userId ? USER_JOINED_LINK(userId) ;
  }

Reply:
fixed this